sdk/android/api/org/webrtc/EglBase.java

```diff
@@ -152,8 +153,9 @@ static int getOpenGlesVersionFromConfig(int[] configAttributes) {
    */
   public static EglBase create(@Nullable Context sharedContext, int[] configAttributes) {
     if (sharedContext == null) {
-      return EglBase14Impl.isEGL14Supported() ? createEgl14(configAttributes)
-                                              : createEgl10(configAttributes);
+      // RingRTC change (or retention?) to keep support for SDK <= 19
+      return Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1 ? createEgl14(configAttributes)
```

Comment:
Are we sure that this check is the same as EglBase14Impl.isEGL14Supported()?

Reply:
Here is the previous code (now removed):

```
private static final int EGLExt_SDK_VERSION = Build.VERSION_CODES.JELLY_BEAN_MR2;
private static final int CURRENT_SDK_VERSION = Build.VERSION.SDK_INT;

// EGL 1.4 is supported from API 17. But EGLExt that is used for setting presentation
// time stamp on a surface is supported from 18 so we require 18.
public static boolean isEGL14Supported() {
    return (CURRENT_SDK_VERSION >= EGLExt_SDK_VERSION);
}
```

So... I guess it should be MR1 instead of MR2.  Both of those are before Kitkat anyway, so it doesn't matter for us.    But I'll change it to MR1 anyway just in case.